.github/pull_request_template.md

```diff
@@ -31,3 +31,19 @@ Tick the boxes that are relevant to your changes, and delete any items that are
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have checked that new and existing unit tests pass locally with my changes
+
+### Release Notes
+
+<!--
+Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.
+
+For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates.
+-->
+
+- [ ] Protocol:
+- [ ] Nodes (Validators and Full nodes):
+- [ ] gRPC:
+- [ ] JSON-RPC:
+- [ ] GraphQL:
+- [ ] CLI:
+- [ ] Rust SDK:
```

Comment:
NOTE_ORDER array in `release_notes.py` seems out of sync with what we actually define here.

Reply:
I don't think the order matters, from what I can tell

Reply:
I guess @Alex6323 was referring to the missing `gRPC`

Reply:
fixed